### PR TITLE
Update Messages & globals urls to use https

### PR DESCRIPTION
**Pull request by pStyl3**

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -1003,7 +1003,7 @@ globals:
   - <<: *versionXIncY
     subs:
       - '**SKSE Plugin ([ENB Helper SE](https://www.nexusmods.com/skyrimspecialedition/mods/23174/))**'
-      - '[ENBSeries](http://enbdev.com/download_mod_tesskyrimse.html)'
+      - '[ENBSeries](https://enbdev.com/download_mod_tesskyrimse.html)'
```
Comment (sibir-ine) [inline masterlist.yaml:1006]: The ENB site doesn't seem to support HTTPS. Also, that link is outdated. It's `http://enbdev.com/download_mod_tesskyrimse.htm` now.

File: masterlist.yaml [hunk 2]
```diff
@@ -1014,7 +1014,7 @@ globals:
   # ENBSeries
   - <<: *missingRequirementXforY
     subs:
-      - '[ENBSeries](http://enbdev.com/download_mod_tesskyrimse.html)'
+      - '[ENBSeries](https://enbdev.com/download_mod_tesskyrimse.html)'
```
Comment (sibir-ine) [inline masterlist.yaml:1017]: Same issue here.

File: masterlist.yaml (hunk 1 above)
Comment (pStyl3) [reply masterlist.yaml:1006]: I have no problem opening that website using `https`, I checked it before changing it.
Besides that, the url can be updated of course.

Comment (sibir-ine) [reply masterlist.yaml:1006]: It works, it's just technically not correct.

Comment (pStyl3) [reply masterlist.yaml:1006]: I guess you're right. Will change it in another PR.

**Pull request merged**